src/helpers/management_utils.ts

```diff
@@ -229,24 +246,24 @@ export function getCommandFiles(): Promise<{ [commandName: string]: BaseCommand
  * @param command - The Command class
  * @param commandName - The name/alias of the command
  */
-function registerCommand(command: BaseCommand, commandName: string) {
-    if (commandName in state.commands) {
+function registerCommand(command: BaseCommand, commandName: string, reload = false) {
+    if (commandName in state.commands && !reload) {
         logger.error(`Command \`${commandName}\` already exists. Possible conflict?`);
         process.exit(1);
     }
     state.commands[commandName] = command;
 }
 
 /** Registers commands */
-export async function registerCommands() {
+export async function registerCommands(reload = false) {
```

Comment:
Reload seems like a bit of a misnomer here, just guessing here but seems like `registerCommands(false)` is only ever called on startup. Might as well assert that commands is empty when not reloading for clarity since it's effectively going to fail anyways.

Maybe im just boomer brain but it took me a second to understand whats going on here